Resources/config/services.yml

```diff
@@ -107,3 +101,17 @@ services:
 
         tags:
             - {name: 'request.param_converter', converter: 'json_api.request_params.converter.http_request_converter'}
+
+    json_api.datetime_handler:
+        class: Mango\Bundle\JsonApiBundle\Serializer\Handler\DateHandler
+        arguments:
+            - '@jms_serializer.datetime_handler'
+        tags:
+            - { name: jms_serializer.subscribing_handler }
+
+    json_api.array_collection_handler:
+        class: Mango\Bundle\JsonApiBundle\Serializer\Handler\ArrayCollectionHandler
```

Comment:
Same as above

Reply:
@oleg-andreyev it seems that `getSubscribingMethods` is the only way how to declare format and there no other option to declare `DateHandler` for `json:api`